sasmodels/models/polymer_micelle.py

```diff
@@ -2,15 +2,15 @@
 
 This model provides the form factor, $P(q)$, for a micelle with a spherical
 core and Gaussian polymer chains attached to the surface, thus may be applied
-to block copolymer micelles. To work well the Gaussian chains must be much
-smaller than the core, which is often not the case.  Please study the
-reference carefully.
+to block copolymer micelles. Take special care before use and read both of the 
+provided references carefully; this model typically only works well when the 
+Gaussian chains are much smaller than the core, which is often not the case. 
 
 Definition
 ----------
 
 The 1D scattering intensity for this model is calculated according to
-the equations given by Pedersen (Pedersen, 2000), summarised briefly here.
+the equations given by Pedersen (Pedersen, 1996 and Pedersen, 2000), summarised briefly here.
```

Comment:
We are loosely adhering to an 80 character line length in the models. Maybe add a line break or shorten this to:
```text
the equations given by Pedersen (1996, 2000), summarised briefly here.
```